examples/sample-app/README.md

```diff
@@ -268,18 +265,18 @@ the present working directory is the same directory as this README.
 
     *Note:* you can also get this information from the web console if you launched it in step 9.
```

Comment:
step 8.  again or just remove the exact step to make updating easier in the future.